jobboard/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.conf import settings
+
+
+class Gender(models.TextChoices):
+    MALE = "M", "Male"
+    FEMALE = "F", "Female"
+    UNSPECIFIED = "U", "Unspecified"
+
+
+class Company(models.Model):
+    company_name = models.CharField(max_length=255)
+    company_description = models.TextField(max_length=500)
+    company_website_url = models.CharField(max_length=255)
+
+
+class Recruiter(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+    company_id = models.ForeignKey(Company, on_delete=models.RESTRICT, related_name='%(class)s_recruiters')
+    email = models.EmailField(max_length=255)
+    gender = models.CharField(max_length=1, choices=Gender.choices, default=Gender.UNSPECIFIED)
+    contact_number = models.IntegerField()
+
+
+class Student(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+    email = models.EmailField(max_length=255)
+    date_of_birth = models.DateField()
+    gender = models.CharField(max_length=1, choices=Gender.choices, default=Gender.UNSPECIFIED)
+    contact_number = models.IntegerField()
```

Comment:
Is this a phone number? 

https://github.com/google/libphonenumber/blob/master/FALSEHOODS.md

Reply:
Got it, I'll change it to a CharField as I think using a 3rd party library is a bit overdoing it as we're not really going to validate the phone numbers anyway.
Also I'll make it optional (allow blank\null)

Reply:
I did not advocate for the library - its just to make you think about the assumptions that you might be making when you choose a particular type.